storybook/src/pages/amopis/common/amopis.css

```diff
@@ -0,0 +1,62 @@
+.ams-docs-amopis-app-grid {
+  display: grid;
+  grid-template-columns: 112px 1fr;
+}
+
+.ams-docs-amopis-app-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.ams-docs-amopis-main-background {
+  background-color: var(--ams-color-neutral-grey1);
+  flex: auto;
+}
+
+.ams-docs-amopis-white-background {
+  background-color: var(--ams-color-primary-white);
+}
+
+.ams-docs-amopis-sidebar {
+  align-items: center;
+  background-color: var(--ams-color-primary-blue);
+  display: flex;
+  flex-direction: column;
+  padding-block-start: calc(40px + 3 * var(--ams-grid-row-gap-md));
+}
+
+.ams-docs-amopis-tabs-vertical {
+  border-block-end: none;
+  flex-direction: column;
+}
+
+.ams-docs-amopis-tabs-vertical .ams-tabs__button {
+  align-items: center;
+  border-block-end: var(--ams-tabs-list-border-block-end);
+  display: flex;
+  gap: var(--ams-space-xs);
+  padding-inline: var(--ams-space-sm);
+  text-align: start;
+}
+
+@media (width <= 1087px) {
```

Comment:
We decided on using min- and max-width. And also the breakpoints util?

Reply:
Agreed, this is not ready to publish.